packages/calypso-e2e/src/lib/pages/themes-detail-page.ts

```diff
@@ -9,10 +9,6 @@ const selectors = {
 	activateDesignButton: 'button:text("Activate this design")',
 	customizeDesignButton: 'span:text("Customize site")',
 
-	// Activate modal
-	activateModal: '.themes__auto-loading-homepage-modal',
-	activateModalButton: '.dialog__action-buttons button:has-text("Activate")',
-
```

Comment:
**praise** - thanks for not just removing the relevant part of the code but also removing the orphaned selectors.